my_init.d/factory_reset.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+# Debugging.
+set -x
+
+# Environment.
+export SHELL=/bin/bash
+
+# Performing factory reset of your AiiDAlab environment:
+# 0 - No reset (noop).
+# 1 - Remove locally installed software and apps (removes ~/apps/ and ~/.local/).
+# 2 - Remove all files and directories within the users home directory.
+
+
+case "${AIIDALAB_FACTORY_RESET}" in
+
+  0)
+    exit 0
+    ;;
+
+  1)
+    rm -rf "/home/${SYSTEM_USER}/apps"
+    rm -rf "/home/${SYSTEM_USER}/.local"
+    ;;
+
+  2)
+    rm -rf /home/${SYSTEM_USER}/*
```

Comment:
The current implementation will not delete hidden directories.

This should work (from [here](https://unix.stackexchange.com/a/77128)):
```suggestion
    find home/${SYSTEM_USER}/ -mindepth 1 -delete
```